src/Config/Validator/Deploy/DatabaseSplitConnection.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Config\Validator\Deploy;
+
+use Magento\MagentoCloud\Config\Database\DbConfig;
+use Magento\MagentoCloud\Config\Stage\DeployInterface;
+use Magento\MagentoCloud\Config\Validator;
+use Magento\MagentoCloud\Config\Validator\ResultFactory;
+use Magento\MagentoCloud\Config\ValidatorInterface;
+
+/**
+ * Validates the split database connections in DATABASE_CONFIGURATION variable
```

Comment:
The description seems not correct. This validator doesn't validate connections it validates their existence.

Reply:
Done